api/pom.xml

```diff
@@ -6,19 +6,14 @@
     <name>OpenFastTrace API</name>
     <parent>
         <relativePath>../parent/pom.xml</relativePath>
-        <groupId>org.itsallcode</groupId>
+        <groupId>org.itsallcode.openfasttrace</groupId>
```

Comment:
Okay. Works for me. Looks a little bit strange at first glance, but makes sense.